util/parse/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+//! Miscellaneous parsing and formatting utilities
+
+#![deny(missing_docs)]
+
+use core::fmt::Display;
+use itertools::Itertools;
+use std::{convert::TryFrom, fs, str::FromStr, time::Duration};
+
+pub use mc_sgx_css::Signature as CssSignature;
+
+/// Parse a number of seconds into a duration
+///
+/// This can be used with structopt
+pub fn parse_duration_in_seconds(src: &str) -> Result<Duration, std::num::ParseIntError> {
+    Ok(Duration::from_secs(u64::from_str(src)?))
+}
+
+/// Load a CSS file from disk. This represents a signature over an enclave,
+/// and contains attestation parameters like MRENCLAVE and MRSIGNER as well
+/// as other stuff.
+pub fn load_css_file(filename: &str) -> Result<CssSignature, String> {
+    let bytes =
+        fs::read(filename).map_err(|err| format!("Failed reading file '{}': {}", filename, err))?;
+    let signature = CssSignature::try_from(&bytes[..])
+        .map_err(|err| format!("Failed parsing CSS file '{}': {}", filename, err))?;
+    Ok(signature)
+}
+
+/// Helper to format a sequence as a comma-separated list
```

Comment:
I don't feel strongly about this, but this doesn't seem to belong in the parsing module... if it's only used by Ingest, why did it move here?

Reply:
i think i might use it other places, idk, i think this is the most idiomatic way right now to display a comma separated list of items in a container. it was weird to have it be like a formatting utility private to ingest server

i could move it back i guess